.ci/teardown.sh

```diff
@@ -92,6 +96,9 @@ collect_logs()
 
 		[ "${have_collect_script}" = "yes" ] && sudo -E PATH="$PATH" "${collect_script_path}" > "${collect_data_log_path}"
 
+		tracing_log_directory="/var/log/kata-tests/logs/traces/"
+		cat "${tracing_log_directory}/${tracing_log_filename}" > "${tracing_log_path}"
```

Comment:
Q, why not just `cp "${tracing_log_directory}/${tracing_log_filename}" "${tracing_log_path}"`?